integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -183,6 +183,22 @@ def test_shift_right_unsigned(data_gen):
                 'shiftrightunsigned(a, cast(null as INT))',
                 'shiftrightunsigned(a, b)'))
 
+@pytest.mark.parametrize('data_gen', [decimal_gen_scale_precision], ids=idfn)
```

Comment:
`round` and `bround` support all numeric types, not just decimal (float, int, byte, double, etc).  Do we have tests for these too?

Reply:
Yes, will add tests for other numeric types.

Reply:
Added tests for other numeric types.